app/api/dao/user.py

```diff
@@ -148,13 +148,14 @@ def get_user_by_username(username: str):
         return UserModel.find_by_username(username)
 
     @staticmethod
-    def list_users(user_id: int, search_query: str = "", page: int = DEFAULT_PAGE, per_page: int = DEFAULT_USERS_PER_PAGE, is_verified = None):
+    def list_users(user_id: int, search_query: str = "", page: int = DEFAULT_PAGE, per_page: int = DEFAULT_USERS_PER_PAGE, is_verified=None, skills = None):
```

Comment:
```suggestion
    def list_users(user_id: int, search_query: str = "", page: int = DEFAULT_PAGE, per_page: int = DEFAULT_USERS_PER_PAGE, is_verified=None, skills=None):
```

Reply:
Yes, It can search from multiple user skills as long as all skills are inside a string. (Which is true because skills is of type string)
Thanks, @SanketDG for the review.

Reply:
I still don't get it though.

Could you give me an example?

Reply:
Ok @SanketDG 
Because the skills field is free-flow text this is the best thing we can do.
if I search for science it will give me below results.
```
'science' 
'social science' 
'Science & social science'
'science and social science'
'Science, social science'
'Science | Social Science'
'Maths and Science'
```



I think this is what you wanted to handle if that is not the case. Please elaborate.

Reply:
Got you, my question was along the lines of searching for multiple skills that are not in order.

So my input would be `science,maths`, would that print the first and last skills?

Reply:
Hi @SanketDG,

No, It will return not anything from the above example. 
Right now, it is working as a regular string match works. We can search for one skill at a time. 

And this looks fine to me, given that skills is string and we don't have any constraints over it. 

If we want multiple skills search than we should define skills in a better way. 
For example - 
1) Use array for skills 
2) define saperator 
3) define a fix list of skills available etc.

Reply:
> If we want multiple skills search than we should define skills in a better way.
> For example -
> 
> 1. Use array for skills
> 2. define saperator
> 3. define a fix list of skills available etc.

That's what I expected from this PR though. But if you say so!